CMakeLists.txt

```diff
@@ -134,6 +134,14 @@ if (NO_MULTIREGION_TEST)
     message(STATUS "NO_MULTIREGION_TEST is ON")
 endif()
 
+set(RESTART_TEST TRUE)
+option(NO_RESTART_TEST "Disable restart simulation tests" OFF)
```

Comment:
Actually `option` will create a variable directly, so I would guess the set part is not that necessary.